.idea/vcs.xml

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="CommitMessageInspectionProfile">
```

Comment:
What's this for?

Reply:
Turns on length checking for commit subject lines vs body.

Reply:
Well I always commit from the terminal, but I have no objection to you using this if you want.